pages/_app.js

```diff
@@ -29,8 +29,8 @@ export default function App({ Component, pageProps }) {
       <ThemeProvider theme={colors}>
         <Context.Provider value={{ state, dispatch }}>
           <Header>
-            <Logo>Florish</Logo>
-            {(user && <Link href="/api/logout"><a>Logout</a></Link>) || <Link href="/signup"><a>Login</a></Link>}
+            <img src="/static/logo.png" style={{ 'height':'3em' }}/>
+            {(user && <Link href="/api/logout"><a style={{ 'float':'right', 'marginTop':'0.5em' }}>Logout</a></Link>) || <Link href="/signup"><a style={{ 'float':'right', 'marginTop':'0.5em' }}>Login</a></Link>}
```

Comment:
I would've used flexbox instead of float and would have created a separate component for the header, but its not a big deal at this stage. this approach works.

Reply:
noted, let's do that in the future if we ever add more to the header